functions.php

```diff
@@ -7,28 +7,24 @@
  * Version: 4.0.0
```

Comment:
Version will have to be changed in all files (just a reminder)

Reply:
Yes tnx this will be done in the later PR